### DISCUSSION TITLE
--json flag for linter not working

### DISCUSSION THREAD

**Discussion by hekatx** [rome/tools#4567] [Q&A] [answered]
In the CLI, seems like the flag `--json` is supported on `rome format` but it doesn't seem to be working on `rome check` even tho the docs say both commands support the flag, am I missing something? running `rome check [path] --json` prints the same output `rome check [path]` does

**Comment by ematipico** [comment, marked as answer]
The flag was more an experiment for us maintainers, in fact it was never officially announced or tracked in the changelog for that reason, even though it's visible in the CLI help screen (we should hide it).
